app/AndroidManifest.xml

```diff
@@ -426,7 +426,7 @@
         android:name="android.permission.USE_CREDENTIALS" />
 
     <uses-sdk
-        android:minSdkVersion="8"
+        android:minSdkVersion="10"

```

Comment:
We can't change this, unfortunately, our app needs to be able to target SDK 8, and last time we changed this a few projects got blocked.